main.test.js

```diff
@@ -1,38 +1,233 @@
-
-// add jquery to global scope
 global.__DEV__ = true
 global.$ = require('jquery');
+global.d3 = require('d3');
+
+const puppeteer = require('puppeteer');
+const fs = require('fs');
 
-let main;
+describe('Opening viewTree.html', () => {
+  let contentHtml = fs.readFileSync('./viewtree.html', 'utf8');
+  let browser;
+  let page;
+  beforeEach(async () => {
+    browser = await puppeteer.launch();
+    page = await browser.newPage();  
+    // await page.setContent(contentHtml, { waitUntil: "networkidle0" });
+    // await page.addScriptTag({path: "main.js"});
+    await page.goto("http://127.0.0.1:5500/viewtree.html"); // do this to be able to call functions in main.js, might need to direct the browser at this url
```

Comment:
use https://camel-lab.github.io/palmyra/viewtree.html , it works! This way, we test what annotators are using online, and it isn't restricted to your local configuration.